index.js

```diff
@@ -0,0 +1,108 @@
+const section3 = document.getElementById("speakers");
+const hambugermenu = document.querySelector(".hambuger");
+const menu = document.querySelector(".navbar-list1");
+console.log(section3);
+
+hambugermenu.addEventListener("click", () => {
+  menu.classList.add("menuvisible");
+});
+
+const icon = `<i class="fa-solid fa-x"></i>`;
+const li = document.createElement("li");
+li.innerHTML = icon;
+menu.append(li);
+
+const closemenu = document.querySelector(".fa-x");
+
+closemenu.addEventListener("click", () => {
+  menu.classList.remove("menuvisible");
+});
+
+const content = `  <div class="speakers"><h4>Feature Speakers</h4></div>
+    <div class="flex-container">
+      <div class="speakeritems">
+        <img src="./images/speakers/speaker_01.png" alt="" class="img" />
+        <div class="itemsA itemA-1">
+          <h5>Yochai Benkler</h5>
+          <span
+            >Berkman Professor of Entrepreneurial Legal Studies at Harvard Law
+            School</span
+          >
+          <p>
+            Benkler studies commons-based peer production, and published his
+            seminal book, The Wealth of Networks in 2006
+          </p>
+        </div>
+      </div>
+      <div class="speakeritems">
+        <img src="./images/speakers/speaker_02.png" alt="" class="img" />
+        <div class="itemsA ItemA-2">
+          <h5>SohYeong Noh</h5>
+          <span
+            >Director of Art Centre Nabi and a board member of CC Korea</span
+          >
+          <p>
+            As the main venue for new media art production in Korea, Nabi
+            promotes cross-disciplinary collaboration and understanding among
+            science technology, humanities, and the arts.
+          </p>
+        </div>
+      </div>
+
+      <div class="speakeritems">
+        <img src="./images/speakers/speaker_03.png" alt="" class="img" />
+        <div class="itemsA itemA-3">
```

Comment:
- [ ] [OPTIONAL] Taking this speaker description as a reference, I suggest that you ensure that the description of the speaker has a design that matches the original design. In the screen sizes, the text should be responsive to match the design
- [ ] [OPTIONAL] In addition, please ensure that the color of the text description matches the original design.

Your design 😎 
![image](https://github.com/enocol/capstone/assets/99394374/3188a795-d9a2-44a7-b20c-001116413980)

Original design 😎 
![image](https://github.com/enocol/capstone/assets/99394374/8ebb45d8-2566-4d36-be0e-cf77b4ff1e93)